### PR TITLE
[492072] Regenerated Xtend code

### PR DESCRIPTION
**Pull request by kthoms**
Produced with Xtend IDE 2.11.0.v20170102-0850

Signed-off-by: Karsten Thoms <karsten.thoms@itemis.de>

### CODE REVIEW

**Comment by eclipsewebmaster** [comment]
Issue tracker reference:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=492072

**Comment by kthoms** [comment]
Build: http://services.typefox.io/open-source/jenkins/job/xtext-core/job/kth_bug492072_xtendgen/

File: org.eclipse.xtext.ide.tests/xtend-gen/org/eclipse/xtext/ide/tests/server/BuildOrderTest.java [hunk 1]
```diff
@@ -53,9 +53,10 @@ public void testChain() {
       it.setDependencies(Collections.<String>unmodifiableList(CollectionLiterals.<String>newArrayList()));
     };
     final ProjectDescription d = ObjectExtensions.<ProjectDescription>operator_doubleArrow(_projectDescription_3, _function_3);
-    this.assertEquals(Collections.<ProjectDescription>unmodifiableList(CollectionLiterals.<ProjectDescription>newArrayList(d, c, b, a)), new TopologicalSorter().sortByDependencies(Collections.<ProjectDescription>unmodifiableList(CollectionLiterals.<ProjectDescription>newArrayList(a, b, c, d)), ((Procedure1<ProjectDescription>) (ProjectDescription it) -> {
+    final Procedure1<ProjectDescription> _function_4 = (ProjectDescription it) -> {
```
Comment (holgerschill) [inline org.eclipse.xtext.ide.tests/xtend-gen/org/eclipse/xtext/ide/tests/server/BuildOrderTest.java]: Is there a reason why the compiler now introduces this variable? Is that by intention?

File: org.eclipse.xtext.ide/xtend-gen/org/eclipse/xtext/ide/server/ServerLauncher.java [hunk 2]
```diff
@@ -38,15 +37,15 @@
   private static boolean IS_DEBUG = false;
   
   public static void main(final String[] args) {
-    ServerLauncher.IS_DEBUG = IterableExtensions.<String>exists(((Iterable<String>)Conversions.doWrapArray(args)), ((Function1<String, Boolean>) (String it) -> {
```
Comment (holgerschill) [inline org.eclipse.xtext.ide/xtend-gen/org/eclipse/xtext/ide/server/ServerLauncher.java]: Is this done by intention?

File: org.eclipse.xtext.ide/xtend-gen/org/eclipse/xtext/ide/server/WorkspaceManager.java [hunk 3]
```diff
@@ -193,27 +181,17 @@ public ProjectManager getProjectManager(final String projectName) {
...
-      WorkspaceManager.LOG.error((("The document " + uri) + " has not been opened."));
-      return;
-    }
-    final Document contents = this.openDocuments.get(uri);
-    Document _applyChanges = contents.applyChanges(changes);
-    this.openDocuments.put(uri, _applyChanges);
-    ArrayList<URI> _newArrayList = CollectionLiterals.<URI>newArrayList();
-    this.doBuild(Collections.<URI>unmodifiableList(CollectionLiterals.<URI>newArrayList(uri)), _newArrayList, cancelIndicator);
+  public void didChange(final URI uri, final int version, final /* Iterable<TextEdit> */Object changes, final CancelIndicator cancelIndicator) {
+    throw new Error("Unresolved compilation problems:"
```
Comment (holgerschill) [inline org.eclipse.xtext.ide/xtend-gen/org/eclipse/xtext/ide/server/WorkspaceManager.java]: This looks like a bug.

Comment (kthoms) [inline org.eclipse.xtext.ide/xtend-gen/org/eclipse/xtext/ide/server/WorkspaceManager.java]: Yes indeed. Please search for such cases, I'll do the same and update the PR.

Comment (kthoms) [inline org.eclipse.xtext.ide/xtend-gen/org/eclipse/xtext/ide/server/WorkspaceManager.java]: Seems it was the only location.

File: org.eclipse.xtext.ide.tests/xtend-gen/org/eclipse/xtext/ide/tests/server/BuildOrderTest.java (hunk 1 above)
Comment (kthoms) [inline org.eclipse.xtext.ide.tests/xtend-gen/org/eclipse/xtext/ide/tests/server/BuildOrderTest.java]: I'm not sure what causes this, but actually I find it better when closures are not inlined. So actually the new result looks better to me. What do you think?

**Pull request merged**